internal/cmd/ske/credentials/start-rotation/start_rotation.go

```diff
@@ -0,0 +1,110 @@
+package startrotation
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/ske/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/spinner"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske/wait"
+)
+
+const (
+	clusterNameArg = "CLUSTER_NAME"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ClusterName string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("start-rotation %s", clusterNameArg),
+		Short: "Starts the rotation of the credentials associated to a SKE cluster",
+		Long: fmt.Sprintf("%s\n%s\n%s",
+			"Starts the rotation of the credentials associated to a STACKIT Kubernetes Engine (SKE) cluster.",
+			"This is step 1 of a two-step process, complete the rotation by running:",
```

Comment:
I think it can be a little more eye-catching this way

```suggestion
			"This is step 1 of a 2-step process, complete the rotation by running:",
```

Reply:
But there is already a nice description in the Portal, we could try to replicate it (or some part) here

Reply:
<img width="529" alt="image" src="https://github.com/stackitcloud/stackit-cli/assets/28832884/32649b8d-a215-4b16-8c5b-5b036fcb68b6">